exercises/darts/test/darts_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:test/test.dart';
+import 'package:darts/darts.dart';
+
+void main() {
+  final darts = new Darts();
+
+  group('Darts', () {
+    group('Return the correct amount earned by a dart landing in a given point in the target problem.', () {
+      test("A dart lands outside the target", () {
+        final num result = darts.score(-9, 9);
```

Comment:
Since the score function returns an `int` I recommend dropping the use of `num` and replacing it with `int`

Reply:
Sounds good, and I can make this change when I get home today. :) One thing I want to point out, this file is created by `bin/create_exercise.dart` so I wasn't sure if it was ok to change the generated file or not.

Reply:
Oh yeah, create_exercise just generates a file based on a template, but we can on this track we can modify the exercise test cases however we want. We could add, subtract, or modify tests.

Also, for other exercises create_exercise may not be able to create the tests accurately (working on that). If a test is expecting an error, then the generated test case would need to be modified to use a Dart exception class. So none of the  generated files from create_exercise are set in stone.